src/notebooks_service.py

```diff
@@ -95,10 +97,28 @@ def health():
 @authenticated
 def whoami(user):
     """Return information about the authenticated user."""
-    return Response(
-        json.dumps(user, indent=1, sort_keys=True),
-        mimetype='application/json',
+    headers = {auth.auth_header_name: 'token {0}'.format(auth.api_token)}
+    info = json.loads(
+        requests.request(
+            'GET',
+            '{prefix}/users/{user[name]}'.format(
+                prefix=auth.api_url, user=user
+            ),
+            headers=headers
+        ).text
     )
+    return jsonify(info)
+
+
+# @authenticated
```

Comment:
why is this commented out?

Reply:
Sorry, this was a mistake. I was in a rush to get the code pushed to the server. I fixed this and turned the annotation on.